content/en/synthetics/browser_tests/actions.md

```diff
@@ -1,7 +1,9 @@
 ---
 title: Browser Test Steps
 kind: documentation
-description: Record Steps for a Synthetic Browser Test
+description: Learn how to automatically record and manually set steps in a browser test recording.
+aliases:
+- /synthetics/guide/email-validation
```

Comment:
?

Reply:
I incorporated the Email Validation guide into the Browser Tests doc page.